helix-term/src/commands.rs

```diff
@@ -4594,7 +4590,6 @@ fn paste_impl(
     let mut values = values
         .iter()
         .map(|value| REGEX.replace_all(value, doc.line_ending.as_str()))
-        .map(|value| Tendril::from(value.as_ref()))
```

Comment:
Couldn't you do the `value.as_ref().repeat(count)` here?

Reply:
At first I didn't because I figured it would allocate for all the values, including the ones that don't end up getting used, but now I realize that it wouldn't, since iterators are lazy.

This change will require the above creation of `repeat` also needs to be changed to have the `.repeat` call.